packages/bigtable/src/index.js

```diff
@@ -56,23 +47,47 @@ var PKG = require('../package.json');
  * @param {string|module:compute/zone} options.zone - The zone in which your
  *     cluster resides.
  *
+ * @example
  * //-
- * // <h3>Creating a Cluster</h3>
+ * // <h3>Creating a Compute Instance</h3>
  * //
- * // Before you create your table, you first need to create a Bigtable Cluster
- * // for the table to be served from. This can be done from either the
- * // Google Cloud Platform Console or the `gcloud` cli tool. Please refer to
- * // the <a href="https://cloud.google.com/bigtable/docs/creating-compute-instance">
+ * // Before you create your table, you first need to create a Compute Instance
```

Comment:
It looks like https://cloud.google.com/bigtable/docs/creating-compute-instance is still a useful link. If so, can we link out to it?